### PR TITLE
Highlights and Guidance documentation

### PR DESCRIPTION
**Pull request by VaLiuM09**

### CODE REVIEW

File: docfx/articles/highlights-and-guidance/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Quick Start
+
+You can check out the main features of this add-on in the provided demo scene. After importing the package in a properly set-up VR Builder project, you can access the demo scene from the menu `Tools > VR Builder > Demo Scenes > Highlights and Guidance`. It is necessary to open the demo scene from the menu at least the first time, so a script will copy the required process file in the `StreamingAssets` folder.
+
+// TODO update this
```
Comment (mrwellmann) [inline docfx/articles/highlights-and-guidance/quick-start.md:5]: Needs to be finished

**Review by mrwellmann — CHANGES_REQUESTED** [review]

File: docfx/articles/highlights-and-guidance/quick-start.md (hunk 1 above)
Comment (mrwellmann) [reply docfx/articles/highlights-and-guidance/quick-start.md:5]: @VaLiuM09 this TODO ist still there

Comment (VaLiuM09) [reply docfx/articles/highlights-and-guidance/quick-start.md:5]: Yes I would keep this PR open and finalize it once I've developed this week's feature. There I should describe the demo scene a bit and it's likely to change if we add a new behavior.

**Comment by VaLiuM09** [comment]
Finalized the documentation. I did not setup PDF creation since it might be useless if we change our business model.

**Review by mrwellmann — APPROVED** [review]

**Pull request merged**